code/API_definitions/device-status.yaml

```diff
@@ -779,53 +876,66 @@ components:
             message: "Service unavailable"
   examples:
     ROAMING_STATUS:
-      value:
-        eventSubscriptionId: 123654
-        event:
-          eventType: ROAMING_STATUS
-          eventTime: 2023-01-17T13:18:23.682Z
-          eventDetail:
-            device:
+      value: 
+        id: 123654
+        source: supertelco.notificationSendServer12
+        type: org.camaraproject.device-status.v0.roaming-status
+        specversion: "1.0"
+        datacontenttype: application/json
+        data: 
+          device:
               phoneNumber: 123456789
-            roaming: true
-            countryCode: 208
-            countryName: FR
+          roaming: true
```

Comment:
As per the document, it's mentioned that "The event subscription identifier must have a value when dealing with Resource-based subscriptions." So we should include the "**subscriptionId**" in the callback response if it's relevant or applicable in that context.